redis_test.go

```diff
@@ -159,10 +170,37 @@ func testRedisDBInitialize_NoTLS(t *testing.T, host string, port int) {
 
 }
 
+func testRedisDBInitialize_TLS(t *testing.T, host string, port int) {
+	caCrt := os.Getenv("CACRT")
```

Comment:
Similar comment here, `CA_CERT` might be the more "expected" variable name to set.

Reply:
Great catch. Thank you! I renamed the field from "cacrt" to "ca_crt" to be consistent with other DB secrets engines. But, I did leave this one using camel case instead of snake case because this is also what other DB secrets engines are doing.

Reply:
It references line 24 `CaCrt       string `json:"ca_crt"``